programs/disks/CommandCopy.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "ICommand.h"
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int BAD_ARGUMENTS;
+}
+
+class CommandCopy : public ICommand
+{
+public:
+    CommandCopy() = default;
+    CommandCopy(std::optional<ProgramOptionsDescription> & command_option_description_input)
+    {
+        command_name = "copy";
+        command_option_description.emplace(createOptionsDescription("Help Message for copy", getTerminalWidth()));
+        description = "Recursively copy data containing at `from_path` to `to_path`";
+        usage = "Usage: copy [OPTION]... <FROM_PATH> <TO_PATH>";
+        command_option_description->add(command_option_description_input.value());
+        command_option_description->add_options()
+            ("help,h", "print help message for copy");
+    }
+
+    void execute(
+        const std::vector<String> & command_arguments,
+        const ContextMutablePtr & global_context,
+        const Poco::Util::AbstractConfiguration & config) override
+    {
+        if (command_arguments.size() != 3)
+        {
+            printHelpMessage();
+            throw DB::Exception("Bad Arguments", DB::ErrorCodes::BAD_ARGUMENTS);
+        }
+
+        String disk_name_from = config.getString("diskFrom", config.getString("disk", "default"));
+        String disk_name_to = config.getString("diskTo", config.getString("disk", "default"));
+
+        String path_from = command_arguments[1];
+        String path_to = command_arguments[2];
+        if (path_from[0] == '/')
+            path_from.erase(0, 1);
+        if (path_to[0] == '/')
+            path_to.erase(0, 1);
```

Comment:
why is it needed?

Reply:
I thought the input format of command should looks like command /PATH
disk->getPath() gives the path in the format /PATH/
that's why I erased the first symbol '/'